### PR TITLE
Cli

### LINKED ISSUE #4: Export this as a cli tool

### PR DESCRIPTION
**Pull request by akshayvadher**
Added option to create id via cli. With options `n` to generate n number of ids and `len` to generate ids with length.

Closes #4

### CODE REVIEW

File: cmd/cuid2/main.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+				Value: 1,
+				Usage: "Numbers of ids to generate",
+			},
+			&cli.IntFlag{
+				Name:  "len",
+				Value: 24,
+				Usage: "Length of the Id (between 2 and 32)",
+				Action: func(ctx *cli.Context, v int) error {
+					if v > 32 || v < 2 {
+						return fmt.Errorf("len %v should be between 2 and 36", v)
```
Comment (omjogani) [inline cmd/cuid2/main.go:27]: len %v should be between 2 and **32**

**Review by omjogani — CHANGES_REQUESTED** [review]
- [ ] Change 36 => 32

File: cmd/cuid2/main.go (hunk 1 above)
Comment (akshayvadher) [reply cmd/cuid2/main.go:27]: This is validation, so it is checking reverse. What is the question again?

Comment (akshayvadher) [reply cmd/cuid2/main.go:27]: Oh error message

Comment (akshayvadher) [reply cmd/cuid2/main.go]: Done. Please check

**Review by omjogani — APPROVED** [review]
Nice one!

**Pull request merged**